File: content/en/docs/Tutorials/Navigate the maze/moving2.c
//Inside of main
leftDist = robot->getDistanceSensor("leftDist");    // Grab robot's left distance sensor
leftDist->enable(timeStep);     // Enable the distance sensor

frontDist = robot->getDistanceSensor("frontDist");
frontDist->enable(timeStep);

rightDist = robot->getDistanceSensor("rightDist");
rightDist->enable(timeStep);

cam = robot->getCamera("camera");
cam->enable(timeStep);

colorSensor = robot->getCamera("color");
colorSensor->enable(timeStep);

emitter = robot->getEmitter("emitter");     // Emitter doesn't need enable

gps = robot->getGPS("gps");
gps->enable(timeStep);

File: content/en/docs/Tutorials/Navigate the maze/moving2.py
#Create objects for all robot sensors
leftDist = robot.getDevice("leftDist")      # Get robot's left distance sensor
leftDist.enable(timeStep)     # Enable left distance sensor

frontDist = robot.getDevice("frontDist")
frontDist.enable(timeStep)

rightDist = robot.getDevice("rightDist")
rightDist.enable(timeStep)

cam = robot.getDevice("camera")
cam.enable(timeStep)

colorSensor = robot.getDevice("color")
colorSensor.enable(timeStep)

emitter = robot.getDevice("emitter")    # Emitter doesn't need enable

gps = robot.getDevice("gps")
gps.enable(timeStep)